code/game/machinery/guestpass.dm

```diff
@@ -173,6 +173,8 @@
 					var/area = get_access_desc(A)
 					entry += "[i > 1 ? ", [area]" : "[area]"]"
 			var/obj/item/card/id/guest/pass = new(get_turf(src))
+			if(Adjacent(usr))
+				usr.put_in_hands(pass)
```

Comment:
can you make this use `ui.user` instead of `usr`

Reply:
Done